packages/devtools_app/lib/src/standalone_ui/vs_code/property_editor_panel.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
```

Comment:
Can you add a TODO to add this endpoint to the StandaloneScreenType enum? https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/standalone_ui/standalone_screen.dart/#L19

Reply:
Done, thanks!